store/src/lib.rs

```diff
@@ -236,4 +283,21 @@ impl StoreConfig {
             StoreConfig::Directory { path } => directory::initialize(path),
         }
     }
+
+    pub fn initialize_explicit_mode<OT, K, V, MKT>(
```

Comment:
I see this is for testing actually which makes sense but I'd love if we could still find an elegant solution to this (including abstracting the if/else conditionals on whether xattrs are supported or not) (I'll leave it to you Irene to explore this)